graphql/resolvers.go

```diff
@@ -495,6 +495,13 @@ func (r *mutationResolver) EditSpawnHost(ctx context.Context, editSpawnHostInput
 		opts.DeleteInstanceTags = deletedTags
 	}
 	if editSpawnHostInput.Volume != nil {
+		v, err := r.sc.FindVolumeById(*editSpawnHostInput.Volume)
+		if err != nil {
+			return nil, ResourceNotFound.Send(ctx, fmt.Sprintf("Error finding requested volume id: %s", err))
+		}
+		if v.AvailabilityZone != h.Zone {
```

Comment:
i think the UI doesn't currently have the capability to set the letter identifier in h.Zone